packages/node/src/dev-server.ts

```diff
@@ -305,7 +305,7 @@ function parseRuntime(
     throw new Error(
       `Invalid function runtime "${runtime}" for "${entrypoint}". Valid runtimes are: ${JSON.stringify(
         validRuntimes
-      )}`
+      )}. Learn more: https://vercel.com/docs/concepts/functions`
```

Comment:
This link doesn't mention the runtimes

Reply:
Yeah. It's what @leerob suggested. https://github.com/vercel/vercel/pull/8007#discussion_r907834570

@leerob is this OK? Will that page mention the runtime explicitly or should we link elsewhere?

Reply:
Would be better to use `vercel.link` URL instead, no? So that we can change where it redirects to without having to change CLI code in the future?

Reply:
If the problem is not defining a default export, we have content for that in https://vercel.com/docs/concepts/functions/edge-middleware#creating-edge-middleware. Would be useful to provide the example there rather than pointing to the top level doc.